src/answers/services/answers.service.ts

```diff
@@ -97,8 +97,8 @@ export class AnswersService {
     teams: TeamDocument[];
   }): Promise<IAnswer[]> {
     let filter = {};
-    let teamsManaged = [];
-    const confirmedUsers = [];
+    let teamsManaged: TeamDocument[] = [];
+    const confirmedUsers: (mongoose.Types.ObjectId | string | undefined)[] = [];
```

Comment:
```suggestion
    const confirmedUsers: (mongoose.Types.ObjectId)[] = [];
```

Reply:
This needs to have string and undefined because later on, line 54 adds a string type and line 64 adds an undefined (possible) type to the array

Reply:
String is not required, but undefined is still needed as line 119 adds a possibly undefined type